src/extensions/workflows.ts

```diff
@@ -45,6 +45,12 @@ module.exports = (toolbox: CycliToolbox) => {
       workflowFileName
     )
 
+    if (toolbox.filesystem.exists(target)) {
+      toolbox.interactive.warning(
+        `Workflow file ${workflowFileName} already exists and will be overwritten.`
```

Comment:
I wonder if the workflow isn't something GitHub actions specific, although not sure if it's worth to worry about it now

Reply:
gitlab calls them "pipelines"

Reply:
but I agree we don't have to worry about it now